screenshots/index.js

```diff
@@ -0,0 +1,375 @@
+const puppeteer = require("puppeteer");
```

Comment:
what is this? ANd how to use it? Modify the contributing guide to mention how to use this.

Reply:
Done